include/acl_selectors.php

```diff
@@ -357,8 +357,7 @@ function populate_acl($user = null, $show_jotnets = false) {
 
 			Addon::callHooks('jot_networks', $jotnets);
 		} else {
-			$jotnets .= sprintf(L10n::t('Connectors disabled, since "%s" is enabled.'),
-					    L10n::t('Hide your profile details from unknown viewers?'));
+			$jotnets .= L10n::t('Connectors disabled, since "%s" is enabled.', 'Hide your profile details from unknown viewers?');
```

Comment:
This should actually be `$jotnets .= L10n::t('Connectors disabled, since "%s" is enabled.', L10n::t('Hide your profile details from unknown viewers?'));`

This is piggy-backing off another translation string used in the settings form.

Reply:
I thought I tried this and it yelled at me, I might have had something wrong with a comma or paren.. the new line looks good!

Reply:
Does the new line also work fine with producing the master `messages.po` file? These are two separate strings that do not depend on each other, so they should be translate-able separately.

Reply:
Not, it doesn't, that's why I asked @zeroadam to revert it back to the double `L10n::t` call, which he did lower.